From 84e87688372ac6ef0b5d1259ab78fc3e7e2ed8c9 Mon Sep 17 00:00:00 2001
From: Neal Cardwell <ncardwell@google.com>
Date: Thu, 29 Mar 2018 17:26:59 -0700
Subject: [PATCH 3/6] net-tcp_bbr: add enum names for meaningful indices into pacing_gain[] array

This commit adds human-readable names for meaningful indices into the
pacing_gain[] array.

Signed-off-by: Neal Cardwell <ncardwell@google.com>
Reviewed-by: Yuchung Cheng <ycheng@google.com>
Reviewed-by: Soheil Hassas Yeganeh <soheil@google.com>
Reviewed-by: Priyaranjan Jha <priyarjha@google.com>
---
 net/ipv4/tcp_bbr.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/net/ipv4/tcp_bbr.c b/net/ipv4/tcp_bbr.c
index c029addfac98..ba4f34934b80 100644
--- a/net/ipv4/tcp_bbr.c
+++ b/net/ipv4/tcp_bbr.c
@@ -150,6 +150,11 @@ static const int bbr_drain_gain = BBR_UNIT * 1000 / 2885;
 /* The gain for deriving steady-state cwnd tolerates delayed/stretched ACKs: */
 static const int bbr_cwnd_gain  = BBR_UNIT * 2;
 /* The pacing_gain values for the PROBE_BW gain cycle, to discover/share bw: */
+enum bbr_pacing_gain_phase {
+	BBR_BW_PROBE_UP		= 0,
+	BBR_BW_PROBE_DOWN	= 1,
+	BBR_BW_PROBE_CRUISE	= 2,
+};
 static const int bbr_pacing_gain[] = {
 	BBR_UNIT * 5 / 4,	/* probe for more available bw */
 	BBR_UNIT * 3 / 4,	/* drain queue and/or yield bw to other flows */
-- 
2.17.0.484.g0c8726318c-goog

